### ISSUE TITLE
Uncaught Error: Call to undefined method Friendica\App::getCurlCode()

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#5875]
**I found some fatal errors in PHP error log:**

````
PHP Fatal error:  Uncaught Error: Call to undefined method Friendica\App::getCurlCode() in .../src/Util/HTTPSignature.php:306
Stack trace:
#0 .../src/Protocol/ActivityPub/Transmitter.php(913): Friendica\Util\HTTPSignature::transmit(Array, 'https://mastodo...', 2587)
#1 .../src/Model/Contact.php(1772): Friendica\Protocol\ActivityPub\Transmitter::sendActivity('Follow', 'https://mastodo...', 2587)
#2 .../mod/follow.php(34): Friendica\Model\Contact::createFromProbe(2587, 'https://mastodo...', true)
#3 .../index.php(359): follow_post(Object(Friendica\App))
#4 {main}
  thrown in .../src/Util/HTTPSignature.php on line 306
````

### Friendica version you encountered the problem
2018.12-dev commit b83393070270814a0a943939750b3a2c5564c7cb

### Friendica source (git, zip)
git

### PHP version
7.2.10

### SQL version
15.1 Distrib 10.1.34-MariaDB

**Comment by MrPetovan** [comment]
Caused by #5862
Ping @nupplaphil

**Comment by nupplaphil** [comment]
On way :)

**Comment by MrPetovan** [comment]
Closed via #5876
